<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="DESKTOP-L0MBEAP" name="com.ztrain.web.TestRunner" tests="10" failures="2" timestamp="2022-05-23T18:36:37 CEST" time="179.369" errors="0">
  <testcase name="runScenario" time="0.712" classname="com.ztrain.web.TestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="27.503" classname="com.ztrain.web.TestRunner">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertFalse(Assert.java:78)
at org.testng.Assert.assertFalse(Assert.java:88)
at com.ztrain.web.steps.PaymentValidationSteps.aValidationConfirmationMessageShouldAppearOnTheScreen(PaymentValidationSteps.java:32)
at ✽.A validation confirmation message should appear on the screen(file:///C:/Users/TED/Documents/pawk/formation/selenium%20ISTQB/framework/demo_test/src/test/resources/features/order.feature:55)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase name="runScenario" time="24.824" classname="com.ztrain.web.TestRunner">
    <failure type="java.lang.AssertionError" message="expected:&lt;votre panier à été mis à jour&gt; but was:&lt;Votre panier à été mis à jour&gt;">
      <![CDATA[java.lang.AssertionError: expected:<votre panier à été mis à jour> but was:<Votre panier à été mis à jour>
at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:115)
at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:125)
at com.ztrain.web.steps.AddProductToCartSteps.thenWeObserveAPopPopWhichIndicatesTheFollowingMessageMessage(AddProductToCartSteps.java:36)
at ✽.then we observe a pop pop which indicates the following message votre panier à été mis à jour(file:///C:/Users/TED/Documents/pawk/formation/selenium%20ISTQB/framework/demo_test/src/test/resources/features/order.feature:23)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase name="runScenario" time="23.175" classname="com.ztrain.web.TestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="50.964" classname="com.ztrain.web.TestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="0.542" classname="com.ztrain.web.TestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="24.509" classname="com.ztrain.web.TestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="22.421" classname="com.ztrain.web.TestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="2.550" classname="com.ztrain.web.TestRunner"/>
  <system-out/>
  <testcase name="runScenario" time="2.169" classname="com.ztrain.web.TestRunner"/>
  <system-out/>
</testsuite> <!-- com.ztrain.web.TestRunner -->
